src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -310,6 +356,160 @@ internal override void SetDefaultSourcePath(string defaultPath)
             WriteValueToFile<string>(fileName, "DefaultSourcePath", defaultPath);
         }
 
+#if UNIX
+        /// <summary>
+        /// Gets the identity name to use for writing to syslog.
+        /// </summary>
+        /// <returns>
+        /// The string identity to use for writing to syslog.
+        /// <para>
+        /// The default value is 'powershell'
+        /// </para>
+        /// </returns>
+        internal override string GetSysLogIdentity()
+        {
+            string fileName = Path.Combine(psHomeConfigDirectory, configFileName);
+            string identity = ReadValueFromFile<string>(fileName, "LogIdentity");
+            if
+            (
+                string.IsNullOrEmpty(identity)
+                ||
+                StringComparer.OrdinalIgnoreCase.Compare(LogDefaultValueName, identity) == 0
+            )
+            {
+                identity = "powershell";
```

Comment:
Shall we use `pwsh`? We've changed the name.

Reply:
You just changed the name of the binary.  The product identity seems like it should be `powershell_core` IMO.

Reply:
That makes sense.

Reply:
When discussing this early on, the agreement was to stay with powershell.  Appending _core doesn't add any value on non-windows systems and, on windows, the provider is identified via it's provider guid.

Reply:
But say if I capture a lot of events on a windows system, will it be easy for me to differentiate the powershell core ones from windows powershell ones?

Reply:
Never mind, it's for non-windows logging, so it won't be a problem :)

Reply:
The next PR I submit will clarify that. The provider guid is changed and the location in the event viewer also changes.